exercises/Version-control-basic-set-up-git-R.md

```diff
@@ -28,11 +28,25 @@ before somehow hitting `Ctrl-s` and overwriting all of your hard word.
 Determined to not let this happen again you've committed to using `git` for
 version control.
 
-Install `git` for your operating system following the
+Install `Git` for your operating system following the
 [setup instructions](http://www.datacarpentry.org/semester-biology/computer-setup/). Then
-create a new project for this assignment in RStudio with the following steps:
-
-1. File -> New Project -> New Directory -> Empty Project.
-2. Choose where to put your project.
-3. Select `Create a git repository`.
-4. If everything worked in the upper right corner of RStudio you should see a `Git` tab.
+create a new repo at the Github organization for the class:
+
+1. Navigate to Github in a web browser and login.
+2. Click the `+` at the upper right corner of the page that flashes `Create 
+   new...` and choose `New repository`.
+3. Choose the class organization (e.g., `dcsemesterYEAR`) as the `Owner` of the
+   repo.
+4. Fill in a `Repository name` that follows the form `FirstnameLastname`.
+5. Select `Private`.
+6. Select `Initialize this repository with a README`.
+7. Click `Create Repository`.
+
+Next, set up a project for this assignment in RStudio with the following steps:
+
+1. File -> New Project -> New Directory -> Empty Project
+2. Navigate to your new Git repo -> Click the `Clone or download` button -> Click the `Copy to clipboard` button.
+3. Paste the `Repository URL:`. *The `Project directory name:` should populate*.
```

Comment:
"The `Project directory name:` should populate" -> "A suggested `Project directory name:` should be automatically generated"